app/bt/AffectedUserFlow/AffectedUserContext.tsx

```diff
@@ -16,8 +22,15 @@ export const AffectedUserProvider = ({
 }): JSX.Element => {
   const [code, setCode] = useState('');
 
+  const generateHMACDigest = async () => {
```

Comment:
I wasn't 100% sure where all of this wants to live.  On the one hand this all could live in the `./hmac.ts` and we could lean into treating it more of a service, on the other hand i like have having the application logic independent from the details of the crypto stuff. Especially considering that it's currently unclear if we should be fetching the certificate as soon as possible or only have the user has consented to sharing data.